extras/userdoc/md/documentation/part-1-neurons-and-simple-neural-networks.md

```diff
@@ -75,7 +75,7 @@ One such command is `nest.Models()`, which will return a list of all the
 available models you can use. If you want to obtain more information about a
 particular command, you may use Python’s standard help system.
 
-    nest.Models?
+    nest.Models() 
```

Comment:
Actually, we really want the question mark here, which shows the help for `nest.Models` (`()` would *execute* it). However, because `?` is an IPython built-in, we should probably change the `Python` in line 76 to `IPython`

Reply:
Sorry for interrupting, but, @jessica-mitchell and jougs, you might have been wrong about line 74, because this sentence says `will return a list of all the available models you can use`, which obviously refers to Models(). Still, the next mention of Models (`you may use IPython's standard help system`) at line 78 should indeed be with the question mark.